chore(guides): remove Sonarr v3 guides (#1909)
TRaSH-Guides/Guides: docs/Sonarr/sonarr-v3-eol.md

# Sonarr v3 - End Of Life

> !!! warning "**Sonarr v3 - End Of Life**<br><br>As of December 30th 2023, Sonarr v4 has been officially released.<br>Sonarr v3 is therefore in End of Life status.<br>As v4 has now been released, there will be no further updates to the Sonarr v3 guide.<br>We recommend that all users upgrade to Sonarr v4.<br><center><br>:bangbang: **Before updating to Sonarr v4** :bangbang:<br>it is recommended that you delete any existing Release Profiles</center><br>"

Reviewer: ```suggestion
!!! warning "**Sonarr v3 - End Of Life**<br><br>As of December 30th 2023, Sonarr v4 has been officially released.<br>Sonarr v3 is therefore in End of Life status.<br>As v4 is now the only supported version of Sonarr, we have removed the Sonarr v3 guide.<br>We recommend that all users upgrade to Sonarr v4.<br><center><br>:bangbang: **Before updating to Sonarr v4** :bangbang:<br>it is recommended that you delete any existing Release Profiles</center><br>"
```
